omni-relayer/src/config.rs

```diff
@@ -49,16 +94,47 @@ pub struct Near {
 }
 
 #[derive(Debug, Clone, serde::Deserialize)]
-pub struct Evm {
+pub struct Eth {
     pub rpc_http_url: String,
     pub rpc_ws_url: String,
     pub chain_id: u64,
     pub bridge_token_factory_address: Address,
+    pub light_client: AccountId,
+    #[serde(default = "derive_eth_address_from_private_key")]
+    pub relayer_address: OmniAddress,
+
     pub block_processing_batch_size: u64,
-    pub fin_transfer_gas_estimation: u64,
+}
+
+#[derive(Debug, Clone, serde::Deserialize)]
+pub struct Base {
+    pub rpc_http_url: String,
+    pub rpc_ws_url: String,
+    pub chain_id: u64,
+    pub bridge_token_factory_address: Address,
+    #[serde(default = "derive_base_address_from_private_key")]
+    pub relayer_address: OmniAddress,
 
-    #[serde(default = "derive_evm_address_from_private_key")]
-    pub relayer_address_on_eth: OmniAddress,
+    pub block_processing_batch_size: u64,
+}
+
+#[derive(Debug, Clone, serde::Deserialize)]
+pub struct Arb {
```

Comment:
It would be great to have a single EvmConfig and a mapping from ChainKind to the config. Right now, the configs are identical.

Reply:
Can you please elaborate a little bit on it? How it would look like in a toml file? We can't have a duplicate keys

Reply:
I was able to come up with something like this https://github.com/Near-One/omni-bridge/pull/146/commits/68d8e84b5dae8e48514def5d5ac7e8b1df281dcd